linux/src/main/java/com/github/weisj/darkmode/platform/linux/gnome/GnomeThemeMonitorService.java

```diff
@@ -0,0 +1,82 @@
+package com.github.weisj.darkmode.platform.linux.gnome;
+
+import com.github.weisj.darkmode.platform.ThemeMonitorService;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.github.weisj.darkmode.platform.linux.gnome.GtkVariants.guessFrom;
```

Comment:
Not a big fan of static imports for constants and methods.

Reply:
Changed in 4291852587b63532a766237e118d6d112b33fe2b.